addon-test-support/index.js

```diff
@@ -1,9 +1,26 @@
+import Ember from 'ember';
 import QUnit from 'qunit';
 
+const { warn } = Ember;
+
 const SINON = self.sinon;
 
 export function createSandbox() {
   QUnit.config.current.testEnvironment.sandbox = self.sinon = SINON.sandbox.create();
+
+  self.sinon.sandbox = {
+    create() {
+      warn(
+        'Explicitly calling `sinon.sandbox.create()` in conjunction with ember-sinon-sandbox is not recommended. Please use `this.sandbox` available in your tests to access sinon.',
+        true,
+        {
+          id: 'ember-sinon-sandbox'
+        }
+      );
+
+      return self.sinon;
```

Comment:
Can you update to avoid returning `self.sinon` here, and instead capture the newly created sandbox into a variable and return it from closure scope?  The specific issue I am worried about is some async thing with a handle on the prior sinon, and they would get the wrong/newer sandbox with the code as it is now.

```js
let sandbox = SINON.sandbox.create();
QUnit.config.current.testEnvironment.sandbox = self.sinon = sandbox;
sandbox.sandbox = {
  create() {
    // ..snip..
    return sandbox;
  }
}
```

Reply:
Yep absolutely. Seems like a good change.